SELF_ASSESSMENT.md

```diff
@@ -233,7 +233,56 @@ Automated testing is integrated into most repositories via GitHub Actions. These
+| [Slack](https://cloud-native.slack.com/archives/C06F3PEPNBW)                                                                                     | Real-time chats and team collaboration.                                                                                                                     |
+| [Google Group](https://groups.google.com/g/oscal-compass)                                                                                        | Formal email communications. Address is oscal-compass@googlegroups.com.                                                                                     |
+| [GitHub Issues](https://github.com/oscal-compass/community/issues)                                                                               | Tracks development tasks, bugs, and feature requests.                                                                                                       |
+| [GitHub Pull Requests](https://github.com/oscal-compass/community/pulls)                                                                         | Used exclusively for code reviews and internal feedback on contributions.                                                                                   |
+| [Zoom Meetings - Global](https://zoom-lfx.platform.linuxfoundation.org/meeting/91709345128?password=5510325d-895f-4932-a843-df728dc3028d)        | Every other Tuesday starting on April 23, 2024 · 11:00 – 12:00pm ET [(convert to your local time)](https://mytime.io/11am/ET).                              |
+| [Zoom Meetings - APAC Friendly](https://zoom-lfx.platform.linuxfoundation.org/meeting/97945872533?password=abad1bfe-cc9a-49d8-9aa6-99bb469a434d) | Every other following Thursday starting on November 30, 2024 - 05:00 GMT / 16:00 AEDT / 1030 IST [(convert to your local time)](https://mytime.io/5am/GMT). |
+| [Meeting Agenda & Notes](https://docs.google.com/document/d/1XTYM7xnWlIqd-8Nn5-qtgvgk8kH3NSmYle5yZvaS7qs/edit?usp=sharing)                       | Shared notes and agendas for meetings.                                                                                                                      |
+| [Shared Calendar](https://zoom-lfx.platform.linuxfoundation.org/meetings/trestlegrc?view=week)                                                   | Schedule of upcoming meetings and events.                                                                                                                   |
+| [YouTube Channel](https://www.youtube.com/@OSCAL-Compass)                                                                                        | Provides users access to video recordings, tutorials, and meeting recordings.                                                                               |
+
```

Comment:
Youtube channel is also for public/user communication rather than internal communication.